front/components/EmptyCallToAction.tsx

```diff
@@ -14,24 +14,24 @@ export function EmptyCallToAction({
 }: {
   label: string;
   disabled?: boolean;
-  icon?: ComponentType;
+  icon?: ComponentType | null;
```

Comment:
I see you pass `icon={null}` below rather than omitting the prop

IMHO keeping the inteface `icon?: ComponentType` was fine. Second choice would be `icon: Comp | null` . I prefer the 1st because it's already the most used in code and IMHO nicer to read than having `icon={null}`, but won't fight for it.

However allowing both undefined and null seems weird, would choose one or the other